debian/changelog

```diff
@@ -1,3 +1,9 @@
+daos (1.3.0-16) unstable; urgency=medium
+  [ Brian J. Murrell ]
+  * Enable debuginfo package building on SUSE platforms
```

Comment:
Is there be a corresponding debian/* file edit related to this change?

Reply:
No.  This is just to keep the version-release parity between the two distros.